code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1069,7 +1069,7 @@
 	REMOVE_TRAIT(M, TRAIT_STABLELIVER, type)
 
 /datum/reagent/medicine/cordiolis_hepatico
-	name = "Cordiolis Hepatico"
+	name = "Edgelord Sauce"
```

Comment:
eh, this isn't really much of a sauce, I think you can come up with a better name for it